packages/calcite-components/src/components/tree-item/tree-item.tsx

```diff
@@ -368,13 +386,16 @@ export class TreeItem implements ConditionalSlotComponent, InteractiveComponent
           el.matches("a")
         ) as HTMLAnchorElement;
 
+        this.userInteracted = true;
```

Comment:
Looking for a better name here.

Reply:
We should set some guidelines for this as this pattern is becoming more frequent and we have a couple names in place:

* [`userChangedValue` - `input.tsx`](https://github.com/Esri/calcite-components/blob/8a575e451acb6b33d65124988b2e2f7ea8ec8b81/packages/calcite-components/src/components/input/input.tsx#L440) and other input types.
* [`internalColorUpdateContext` - `color-picker.tsx`](https://github.com/Esri/calcite-components/blob/8a575e451acb6b33d65124988b2e2f7ea8ec8b81/packages/calcite-components/src/components/color-picker/color-picker.tsx#L303)

Maybe `userChangedValue` for now since it's more common.